triage/triaging.rst

```diff
@@ -7,32 +7,24 @@ Triaging an Issue
 This section of the devguide documents the :ref:`issue tracker <tracker>` for
 users and developers.
 
-Contributors with the Triager role on the issue tracker can triage issues
-directly without any assistance.
-
 
 Checklist for Triaging
 ======================
 
-* Read the issue comment(s).
-* Review and set classification fields
-    - Title: should be concise with specifics which are helpful to someone
-      scanning a list of issue titles. (Optional, if possible) Add a
-      prefix at the start of the title to indicate the module, e.g. IDLE,
-      doc, or async.
-    - Type
-    - Stage
-    - Components: multiple items may be set
-    - Versions: set if known, leave blank if unsure. Multiple items may be set.
-* Review and set process fields
-    - Status
-    - Superseder
-    - Assignees
-    - Nosy List
-    - Priority
-    - Keywords
-* (Optional) Leave a brief comment about the proposed next action needed. If
-  there is a long message list, a summary can be very helpful.
+* Check that the title is concise with specifics which are helpful to
+  someone scanning a list of issue titles.
```

Comment:
```suggestion
* Check that the title is reasonably concise, while including enough specifics
  so that those scanning the list of issues can quickly identify its topic.
```

* Use clearer, more descriptive and more idiomatic phrasing
* Avoid repetition and awkwardness

To note, right now triagers can't touch issue titles at all, but once python/core-workflow#460 is implemented (which after more positive feedback in one last round of IRL discussion with those involved, I plan to submit to the SC today and they can hopefully review it as soon as Monday), they will be able to do this.